apps/next-evans/lib/residences.ts

```diff
@@ -43,6 +44,29 @@ function deriveMap(positions: Position[]): 'UK' | 'US' | 'Europe' {
   return arrays[0][0];
 }
 
+export function getPrimaryResidence(residences: Residence[]): Residence | null {
```

Comment:
help me remember that we need to dial this fully in, and test it, according to the swift code i showed you, at some point. doesn't need to hold up this PR though